tests/test_core_network.py

```diff
@@ -2696,8 +2701,12 @@ def test_generate_validation_report_with_correct_schedule(correct_schedule):
                                             'number_of_connected_subgraphs': 0},
                                    'bike': {'problem_nodes': {'dead_ends': [], 'unreachable_node': []},
                                             'number_of_connected_subgraphs': 0}},
-            'link_attributes': {'links_over_1km_length': {'number_of': 0, 'percentage': 0.0, 'link_ids': []},
-                                'zero_attributes': {}}},
+            'link_attributes': {'links_over_1000_length': {'number_of': 0, 'percentage': 0.0, 'link_ids': []},
```

Comment:
We've had to change this test, even though its name suggests it is only concerned with the reporting on the schedule, and nothing has changed in that code. This feels like the test telling you that your assertion is too wide. Should this test care at all how the `link_attributes` section of the report looks? That should be tested elsewhere and removed from the equation in this test.